website/app/server.js

```diff
@@ -0,0 +1,30 @@
+const nodeStatic = require('node-static');
+const path = require('path');
+const fileServer = new nodeStatic.Server(path.resolve(__dirname,'../build/AvoDocs'),{ cache: false })
+const sourceServer = new nodeStatic.Server(path.resolve(__dirname,'sources'),{ cache: false })
+const { createServer } = require('http')
+
+details = {
+  port: 8000,
```

Comment:
Does this need to be static or could it dynamically find a free port? Just worried what will happen if this port is already in use

Reply:
Yes good idea

Reply:
Resolved in 010d57d32fc8ff250ce4a6a6e34908e8eb731e33